src/test/java/core/basesyntax/service/impl/StrategyApplierTest.java

```diff
@@ -0,0 +1,111 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.DaoService;
+import core.basesyntax.db.DaoServiceHashMap;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.strategy.SaveStrategy;
+import core.basesyntax.strategy.impl.SaveStrategyBalance;
+import core.basesyntax.strategy.impl.SaveStrategyPurchase;
+import core.basesyntax.strategy.impl.SaveStrategySupply;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class StrategyApplierTest {
+    private static final String DEFAULT_FRUIT = "apple";
+    private static final String ADDITIONAL_FRUIT = "banana";
+    private static final List<FruitTransaction> VALID_DEFAULT_TRANSACTIONS = new ArrayList<>();
+    private static final List<FruitTransaction> VALID_ADDITIONAL_TRANSACTIONS = new ArrayList<>();
+    private static DaoService STORAGE;
+    private static final Map<Operation, SaveStrategy> STRATEGIES_WITHOUT_RETURN =
+            new HashMap<>();
```

Comment:
no need to use upper case - these are not magic numbers or strings

same everywhere